### PR TITLE
AC: Added LabelMe converter to DetectionAnnotation and SegmentationAnnotation

### PR DESCRIPTION
**Pull request by Anna-Mironova**

### CODE REVIEW

**Comment by eaidova** [comment]
Jenkins please retry a build

File: tools/accuracy_checker/accuracy_checker/annotation_converters/README.md [hunk 1]
```diff
@@ -527,6 +527,15 @@ The main difference between this converter and `super_resolution` in data organi
 * `yolo_labeling` - converts object detection dataset with annotation in YOLO labeling format to `DetectionAnnotation`.
   * `data_dir` - path to directory with annotation files in txt format and images.
   * `labels_file` - path to file with labels in txt format (optional).
+* `label_me_detection` - converts dataset obtained using LabelMe Annotation Tool to `DetectionAnnotation`.
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/annotation_converters/README.md:530]: please add link on the tool

File: tools/accuracy_checker/accuracy_checker/annotation_converters/label_me_converter.py [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+    __provider__ = 'label_me_segmentation'
+    annotation_types = (SegmentationAnnotation, )
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'annotations_dir': PathField(is_directory=True, description='Path to directory with annotation files.'),
+            'images_dir': PathField(optional=True, is_directory=True, description='Path to directory with images.'),
+            'masks_dir': PathField(optional=True, is_directory=True,
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/annotation_converters/label_me_converter.py:111]: can be mask encoded by colors?

File: tools/accuracy_checker/accuracy_checker/annotation_converters/label_me_converter.py [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+    __provider__ = 'label_me_detection'
+    annotation_types = (DetectionAnnotation, )
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'annotations_dir': PathField(is_directory=True, description='Path to directory with annotation files.'),
+            'images_dir': PathField(optional=True, is_directory=True, description='Path to directory with images.'),
+            'labels_file': PathField(description='Path to labels file')
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/annotation_converters/label_me_converter.py:34]: what about background?

File: tools/accuracy_checker/accuracy_checker/annotation_converters/README.md (hunk 1 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/accuracy_checker/annotation_converters/README.md]: Done

File: tools/accuracy_checker/accuracy_checker/annotation_converters/label_me_converter.py (hunk 3 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/accuracy_checker/annotation_converters/label_me_converter.py]: Added new parameter

File: tools/accuracy_checker/accuracy_checker/annotation_converters/label_me_converter.py (hunk 2 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/accuracy_checker/annotation_converters/label_me_converter.py:125]: Updated

File: tools/accuracy_checker/accuracy_checker/annotation_converters/README.md [hunk 4]
```diff
@@ -533,6 +533,16 @@ The main difference between this converter and `super_resolution` in data organi
 * `yolo_labeling` - converts object detection dataset with annotation in YOLO labeling format to `DetectionAnnotation`.
   * `data_dir` - path to directory with annotation files in txt format and images.
   * `labels_file` - path to file with labels in txt format (optional).
+* `label_me_detection` - converts dataset obtained using [LabelMe](http://labelme.csail.mit.edu/Release3.0/) Annotation Tool to `DetectionAnnotation`.
+  * `annotations_dir` - path to directory with annotation files in xml format.
+  * `dataset_meta_file` - path to json file with dataset meta (e.g. label_map, color_encoding). More details in [Customizing dataset meta](#customizing-dataset-meta) section.
+  * `images_dir` -path to directory with images (Optional).
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/annotation_converters/README.md:539]:
```suggestion
  * `images_dir` - path to directory with images (Optional).
```

**Review by eaidova — APPROVED** [review]

**Pull request merged**